### ISSUE TITLE
Does finetune/run_simcse.py only support unsupervised training?

### ISSUE THREAD

**Issue by Eric8932** [dbiir/UER-py#312]
During training, the tgt of the loss function is torch.arange(batch_size), which is suitable for unsupervised training.
But this will overwrite the labels of the supervised training set.

**Comment by ydli-ai** [comment]
For now, we only implemented the unsupervised part of simcse.
If you are interested, you can contribute your code and open a PR.
